### ISSUE TITLE
Nvidia FleX demo returns Cuda error: 2

### ISSUE THREAD

**Issue by PvtNewb** [vosen/ZLUDA#81]
Hardware: RX Vega 7 (Cezanne) with driver 31.0.23013.5003, 4GB UMA allocated out of 32GB of memory.
OS: Windows 11 build 22631
Software tested: [Nvidia FleX PhysX demo](https://github.com/NVIDIAGameWorks/FleX)

After launching with `zluda.exe -- NvFlexDemoReleaseCUDA_x64.exe`, the program would promptly launch for a bit, then crashes with `Flex: Cuda error: 2 in cudaMallocHost(&ptr, elementCount*elementByteStride) - ..\..\cuda\util.cpp:477`.
![image](https://github.com/vosen/ZLUDA/assets/78315800/a6e1fddf-648e-4fd3-bea1-fbaced2397fa)

**Comment by vosen** [comment]
Hmmm, do you have integrated gpu installed? If so see the section about it in the README

**Comment by PvtNewb** [comment]
Yes, I only have the Vega iGPU on my system, well I guess this is one of those GPU ops limitations.

**Comment by DGdev91** [comment]
I can confirm it works on a 7900xt on Windows (slow and with some non-blocking cuda errors, but it runs). It's most likely because of your gpu.

Anyway, if someone wants to debug the error, it' this one. it appears 6 times.
`Flex: Cuda error: 71 in cudaGraphicsResourceSetMapFlags(res, cudaGraphicsMapFlagsWriteDiscard) - ../../cuda/flex.cu:4580`
